### DISCUSSION TITLE
Ranger-style selection in lf?

### DISCUSSION THREAD

**Discussion by hectorBrown** [gokcehan/lf#1100] [Q&A] [answered]
Hi, started using lf as a faster alternative to ranger recently, have been loving it. One thing I miss though is ranger's vi-style "visual mode" for selection. The default in lf seems kind of slow to me, there's no easy way to select lots of files, but I'm not sure how I'd go about binding key to do something like this, or if it's even possible.
Any help is appreciated.
Thanks!

**Comment by ilyagr** [comment, marked as answer]
This is tracked in https://github.com/gokcehan/lf/issues/477. I just had a thought I'll add there.
